cloudera-integration/csd/scripts/control.sh

```diff
@@ -40,6 +40,24 @@ export LOGBACK_CONFIG=${LOGBACK_CONFIG:-$DEFAULT_LOGBACK_CONFIG}
 export KEYTAB_FILE=${KEYTAB_FILE:-"${CONF_DIR}/pulse.keytab"}
 export AKKA_CONF=${AKKA_CONF:-"application.conf"}
 
+if [ -f COLLECTION_ROLLER_CONFIG ]; then #Checking if file exists
```

Comment:
Can you AND these two conditions together?

Reply:
@afoerster  So currently I check:
1 If the file exists?
1.1 If it does then is it non-empty?
1.1.1 If it is non-empty then Use the default config

1.2 Else use config

If I combine the 2 conditions into an AND condition, then we'll be executing step 1.2 without being sure if the file exists(step 1)

Reply:
```
if (file exists and is not empty)
  use it
else
  use the default
```

Does this not work?